### DISCUSSION TITLE
Unable to connect to load balancer IP from web admin portal

### DISCUSSION THREAD

**Discussion by sujit-kamthe** [finos/TimeBase-CE#264] [Q&A]
Hello Team,

I have 2 servers in which i have configured Timebase application & load balancer is configured in front of those 2 servers.
Application is accessible using that LB IP on port 8011
http://<IP or domain name of LB>:8011

Then I configured Timebase Web admin on docker container using below command, 
docker run --rm -d \ 
    --name timebase-admin \
    --network timebase-network
    -p 8099:8099 \ 
    -e "JAVA_OPTS=-Dtimebase.url=dxtick://timebase-server:8011" \
    --ulimit nofile=65536:65536 \
    epam/timebase-ws-server:1.0

Here in-place of "timebase-server" I gave IP address of Load balancer
Then container is started & also Timebase Web admin page is loading on port 8099
http://<IP or domain name of LB>:8099

But at the time of login, I am getting below error,
Cannot open timebase [dxtick://<IP of load balancer>:8011]

I have also attached error in the request.

Need help regarding , how Timebase web admin can able to connect to Timebase server which is behind load balancer?

**Comment by alex-karpovich** [comment]
Greeting @sujit-kamthe 
I assume you have to change web admin connection url to TimeBase here:

```  -e "JAVA_OPTS=-Dtimebase.url=dxtick://IP of load balancer>:8011" ```

**Reply by sujit-kamthe** [reply]
But if I mention timebase url in web admin then how i can achieve high availability.
I have 2 servers & i want HA of Timebase application, that's why i configured LB & mentioned it's IP in web admin.

**Reply by ankitsoin** [reply]
Hi @alex-karpovich 

Is it possible to achieve HA using above design where LB is sitting infront of 2 separate servers containing TB server each?
Also, is it possible to connect web admin with same approach?
